packages/ui/src/containers/DeployContractsContainer.ts

```diff
@@ -15,30 +17,21 @@ import {
   validateInt,
   valueRequired
 } from '../lib/FormsValidator';
-import $ from 'jquery';
-import { Deploy } from 'casperlabs-grpc/io/casperlabs/casper/consensus/consensus_pb';
-import {
-  CLType,
-  Key
-} from 'casperlabs-grpc/io/casperlabs/casper/consensus/state_pb';
-import { decodeBase64 } from 'tweetnacl-ts';
-import JSBI from 'jsbi';
-import { publicKeyHashForEd25519 } from './AuthContainer';
 import { DeployArgumentParser } from '../lib/DeployArgumentParser';
-import MetricsService from '../services/MetricsService';
 import AuthService from '../services/AuthService';
+import { SimpleType, AccessRights } from 'casperlabs-sdk';
+import { decodeBase64 } from 'tweetnacl-ts';
+import MetricsService from '../services/MetricsService';
+import JSBI from 'jsbi';
 
 export const BytesTypeStr = 'Bytes';
 export const BytesFixedTypeStr = 'Bytes (Fixed Length)';
 
-export type SimpleType =
-  | 'Bytes'
-  | 'Bytes (Fixed Length)'
-  | CLType.SimpleMap[keyof CLType.SimpleMap];
+export type UISimpleType = 'Bytes' | 'Bytes (Fixed Length)' | SimpleType;
 
 export type ComplexType = 'Tuple' | 'Map' | 'List' | 'FixedList';
 
-export type SupportedType = SimpleType | ComplexType;
+export type SupportedType = UISimpleType | ComplexType;
```

Comment:
Why UISimpleType not SimpleType ?

Reply:
UISimpleType adds two more types, `Bytes` and `Bytes(fixed length)`. Actually, `CLValue` doesn't support these two types directly, and the internal representation of `Bytes` is `List<u8>`, but in frontend, we provide these types to users so that users could create `Bytes` by a hex-encoded string.